doc/changelog.md

```diff
@@ -69,6 +69,9 @@
 
 <h3>Improvements</h3>
 
+* Catalyst now checks that devices (new API), have the wires set.
+  [(#664)](https://github.com/PennyLaneAI/catalyst/pull/664)
```

Comment:
This could also be added to the existing entry for the new device api, since it's not an improvement on an existing feature (existing as of last release). Not a big deal though.

Reply:
Yes I will move it

Reply:
Done